### ISSUE TITLE
FInetuning the whole unet

### ISSUE THREAD

**Issue by Pang-Yatian** [snap-research/HyperHuman#6]
Hi, thanks for your great work. Please allow me to ask a question that might not relate closely to the paper.

In the paper, you finetune the whole unet with three expert modules, which results in high computational cost. Is it possible to only finetune the extra expert modules(normal and depth) if we assume the rgb branch is already well-trained? Could you please provide some insights on this? Thank you in advance.

**Comment by alvinliu0** [comment]
Hi, many thanks for your interest in our work!

1. As we haven't run such an experiment, I'm not sure if it's OK to only finetune the replicated network layers. Intuitively, I think we should at least finetune **some** of the shared backbone layers. Otherwise, the intermediate features can not be updated to simultaneously output aligned RGB, depth, and normal.

2. There are certain ways to decrease the computational cost for your information. For example, a) change into a smaller backbone to trade-off the performance and memory cost, e.g., Small SD and Tiny SD. b) Use the efficient parameter finetuning techniques like LoRA, or just finetune some certain layers in the shared backbone. c) common engineering tricks to reduce memory consumption as used in diffusers, e.g., gradient checkpointing, gradient accumulation w/ smaller batch size, deepspeed model parallelism, lower fp precision like fp16, efficient_xformers, etc.

Best

**Comment by Pang-Yatian** [comment]
Thanks a lot for your comments.
